test/test_loader.py

```diff
@@ -0,0 +1,91 @@
+import chess
+import numpy as np
+import torch.multiprocessing as mp
+from collections import deque
+import pickle
+import unittest
+import torch
+
+import os,sys,inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0,parentdir)
+
+from loader import Preprocessor, FENGenerator, PGNIterator, StandardConvSuite 
+
+path_to_test_pgn = f"lichessTestPgn.pgn"
+path_to_test_fen = f"lichessTestFen.pgn"
+path_to_test_array = f"lichessTestArray.dictionary"
+path_to_test_tensor = f"lichessTestTensor.dictionary"
+
+
+class testIterator:

```

Comment:
In my opinion it would be more clear if this class inherited from `PGNIterator` and had name starting with uppercase letter

Reply:
I'm not sure about class inheritance, becauseTestIterator has nothing to do with PgnIterator, it just forwards data from generator